format/postgres/common/pg_control.go

```diff
@@ -0,0 +1,135 @@
+package common
+
+import (
+	"fmt"
+	"github.com/wader/fq/pkg/scalar"
```

Comment:
You can run the same lint locally with `make lint`, i also use it with vscode.

Try to follow the linter but if you really don't agree you can add a `//nolint:<linter-name>` line before the affected line to ignore it. Possible also a comment why.